uuid.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+"""Generate a uuid and copy to clipboard"""
+
+import uuid
+
+from albertv0 import *
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "UUID Generator"
+__version__ = "1.0"
+__trigger__ = "uuid"
+__author__ = "Josh Crank"
+__dependencies__ = []
+
+iconPath = iconLookup('preferences-system-privacy')
```

Comment:
This icon is confusing in some icon themes.
I assume it was picked from Adwaita, since it represents a numeric lock.